article.py

```diff
@@ -46,7 +46,7 @@ def save_raw(self):
                       indent=4,
                       ensure_ascii=False,
                       separators=(',', ': '))
-    
+
```

Comment:
discard changes

Reply:
this change was made to get rid of the lint "trailing whitespace" issue

Reply:
still needs to be addressed